mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -160,6 +160,9 @@ for more information.
     this cluster. Note that when this option is enabled, the cluster cannot be upgraded
     and will be automatically deleted after 30 days.
 
+* `enable_k8s_beta_apis` - (Optional) List of Kubernetes beta features to enable for
```

Comment:
This would change to `(Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html))`, to include that field is only available on the beta provider.